robot/Cumulus/tests/browser/gift_entry/verify_mapped_fields_tmpbldr.robot

```diff
@@ -0,0 +1,98 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/AdvancedMappingPageObject.py
+...             robot/Cumulus/resources/ObjectMangerPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable Gift Entry
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+${template}       AM Template
+
+*** Keywords ***
+Setup Test Data
+    Validate And Create Required CustomField
+    ...                                                    Object=Opportunity
+    ...                                                    Field_Type=Text
+    ...                                                    Field_Name=Test Mapping
+    Validate And Create Required CustomField
+    ...                                                    Object=NPSP Data Import
+    ...                                                    Field_Type=Text
+    ...                                                    Field_Name=Opportunity Test
+    ${ns} =  Get NPSP Namespace Prefix
+    Set suite variable    ${ns}
+
+*** Test Cases ***
+
+Verify Mapped Field Is Available For Batch Template
+    [Documentation]                           Create a custom field on opportunity and npsp data import objects. Create advanced mapping to these fields.
+    ...                                       Verify that the mapped field is available for selection while creating a new template and once selected, its avialble on newly created batch gift form
+    ...                                       Remove field from template, delete mapping and verify that the new field is not avaialable for selection while creating template.  
+    [tags]                                    unstable      feature:GE                    W-039562   
+    #Create field mapping
+    Click Configure Advanced Mapping
+    View Field Mappings Of The Object         Opportunity
+    Create Mapping If Doesnt Exist            Opportunity Test (Opportunity_Test__c)    Test Mapping (Test_Mapping__c)
+    Reload Page
+    #Create new template with new field                            
+    Go To Page                                Landing                         GE_Gift_Entry         tab=Batches
+    Click Link                                Templates
+    Click Gift Entry Button                   Create Template
```

Comment:
doesn't click object button not work here?

Reply:
no, thats why i have a new keyword that works for most of gift entry buttons